ksql-functional-tests/src/test/resources/query-validation-tests/create-struct.json

```diff
@@ -0,0 +1,61 @@
+{
+  "comments": [
+    "Tests covering the use of the array returning UDFs."
+  ],
+  "tests": [
```

Comment:
Can we add some negative test cases? e.g. what happens if there are two fields with the same name, or some name but different case? etc. etc.

Also, a test for case-sensitive field names?

Reply:
I guess the tests are awkwardly split between files - i'll add more here